src/System.Text.Json/tests/WritableJsonApiTests.cs

```diff
@@ -625,5 +660,187 @@ public static void TestAquiringAllProperties()
 
             IEnumerable<JsonNode> fullTimeEmployees = employees.GetAllProperties("FTE");
         }
+
+        /// <summary>
+        /// Transforming JsoneNode to JsonElement
+        /// </summary>
+        [Fact]
+        public static void TestTransformingJsonNodeToJsonElement()
+        {
+            // Send Json through network
+            var employeeDataToSend = EmployeesDatabase.GetNextEmployee().Value;
+            Mailbox.SendEmployeeData(employeeDataToSend.AsJsonElement());
+
+            // Send Json that is lightweight and immutable
+            Mailbox.SendEmployeeData(JsonDocument.Parse(employeeDataToSend).RootElement);
+        }
+
+        /// <summary>
+        /// Transforming JsonElement to JsoneNode
+        /// </summary>
+        [Fact]
+        public static void TestTransformingJsonElementToJsonNode()
+        {
+            var receivedEmployeeData = JsonElement.DeepCopy(Mailbox.RetrieveEmployeeData());
+            if (receivedEmployeeData is JsonObject employee)
+            {
+                employee["name"] = new JsonString("Bob");
+                Mailbox.SendEmployeeData(employee.AsJsonElement());
+            }
+        }
```

Comment:
I didn't really notice last time, so I don't know if this is new for this PR or was also true previously, but you're defining `[Fact]`s with no `Assert`s in them... so all it'll test is that nothing throws exceptions.

Reply:
Yes, they aren't tests yet, just scenarios, I'll add all of the asserts later.

Reply:
I added assertions to existing scenarios. I will also add new tests (focusing on single methods).